pkg/plugin/v3/webhook.go

```diff
@@ -74,6 +78,7 @@ func (p *createWebhookSubcommand) BindFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&p.resource.WebhookVersion, "webhook-version", defaultWebhookVersion,
 		"version of {Mutating,Validating}WebhookConfigurations to scaffold. Options: [v1, v1beta1]")
 
+	fs.BoolVar(&p.runMake, "make", true, "if true, run make after generating files")
```

Comment:
Is this required for webhook changes?

Reply:
The initial idea here is to keep it == resources. See `store webhooks config info and add make run (go/v3-alpha)`
However, I agree that it might better be done in another PR.